modules/apps/layout/layout-reports-web/test/js/components/error-alert/ErrorAlert.test.js

```diff
@@ -206,6 +206,11 @@ describe('Error Alert', () => {
 			expect(
 				getByText('this-page-cannot-be-audited')
 			).toBeInTheDocument();
+			expect(
```

Comment:
I usually adapt tests in a different commit, but I guess it's up to you if you prefer doing it this way 😄

Reply:
Uhmm... I add the cases here as they check for the string added in the commit, seemed related to me and the commit was small... I think twice next time, if you agree

Reply:
Sure! It's okay if you do it that way 🥰